.travis.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
see https://github.com/linux-system-roles/network/pull/108/files#r278538735

Reply:
resolved this.